core/query/src/main/java/org/eclipse/rdf4j/query/impl/BackgroundGraphResult.java

```diff
@@ -0,0 +1,244 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.query.impl;
```

Comment:
Why is this class in `oer.query.impl`, but its companion in `oer.query.resultio.helpers`? Shouldn't they both be in the `helpers` package?